.vscode/settings.json

```diff
@@ -11,6 +11,34 @@
         "**/.classpath": true,
         "**/.project": true,
         "**/.settings": true,
-        "**/.factorypath": true
-    }
-}
+        "**/.factorypath": true,
+        },
+        "editor.tokenColorCustomizations":{
+            "comments": "#00aeff",
+            //"variables": "#00eeff"
+        },
+        "workbench.colorCustomizations": {

```

Comment:
This is the problem with doing `git add -a` or `git add *` :grinning:

I'd recommend `git add src` if you're playing around with your settings.